### DISCUSSION TITLE
Trigger "enter" earlier

### DISCUSSION THREAD

**Discussion by axeldvp** [chartjs/chartjs-plugin-annotation#463] [Ideas]
Hello,

I'm actually using annotations to show date intervals, so I'm using large borders. Sometimes intervals are short, so I display labels only on enter.

[Here is an example](https://codepen.io/axel-d/pen/eYRjvre)

Would it be possible to trigger "enter" earlier for an annotation with large borders ? The problem is the cursor needs to be really close to the center of the line to trigger the event.

**Comment by kurkle** [comment]
Would just need to consider `borderWidth` here:

https://github.com/chartjs/chartjs-plugin-annotation/blob/25a195835ac928ab0c26ebfd46b69f90f6bd9cee/src/types/box.js#L6-L13

**Comment by axeldvp** [comment]
Thanks for your answer.

So it's not possible with line annotation,  the only way is to use box annotation ?

**Comment by stockiNail** [comment]
@axeldvp I think your need could be addressed by PR #591 

You can also use a box annotation with the label

**Comment by axeldvp** [comment]
@stockiNail I just tested, it works pretty good, thanks !
